### PR TITLE
Bump API extension from v1beta1 to v1

### PR DESCRIPTION
**Pull request by guangbochen**
**Problem:**
need to bump k8s apiextension from v1beta1 to v1 due to:
```
apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

**Solution:**
Bump API extension from v1beta1 to v1

**Related PR:**
https://github.com/rancher/wrangler/pull/154

### CODE REVIEW

File: pkg/schema/converter/crd.go [hunk 1]
```diff
@@ -88,8 +85,8 @@ func forVersion(crd *beta1.CustomResourceDefinition, group, version, kind string
 		attributes.SetColumns(schema, columns)
 	}
 
-	if crd.Spec.Validation != nil && crd.Spec.Validation.OpenAPIV3Schema != nil {
-		if fieldsSchema := modelV3ToSchema(id, crd.Spec.Validation.OpenAPIV3Schema, schemasMap); fieldsSchema != nil {
+	if crd.Spec.Versions[0].Schema != nil && crd.Spec.Versions[0].Schema.OpenAPIV3Schema != nil {
+		if fieldsSchema := modelV3ToSchema(id, crd.Spec.Versions[0].Schema.OpenAPIV3Schema, schemasMap); fieldsSchema != nil {
```
Comment (gitlawr) [inline pkg/schema/converter/crd.go:89]: Use OpenAPIV3Schema of each version instead of the first one

**Comment by guangbochen** [comment]
@gitlawr updated

**Review by gitlawr — APPROVED** [review]
LGTM

**Comment by StrongMonkey** [comment]
@guangbochen Sorry I didn't see that you already did a PR. This pr is merged https://github.com/rancher/steve/pull/17/files

**Comment by guangbochen** [comment]
@StrongMonkey thanks for the update, as long as this change gets merged it is awesome.

**Pull request closed without merging**